apps/studio/src/features/dashboard/components/FolderSettingsModal/FolderSettingsModal.tsx

```diff
@@ -158,7 +159,12 @@ const SuspendableModalContent = ({
                     placeholder="This is a url for your folder"
                     {...field}
```

Comment:
issue: should we also add the maxLength here?

Reply:
actually i tried this before, but having the controller and the maxlength on the input doesn't quite work oddly enough, which is whyi had to slice

Reply:
wtf i just retried and it works. idk what changd from last time vs now but ok ill do the max length and remove the slice